mesonbuild/wrap/wrap.py

```diff
@@ -695,7 +696,8 @@ def get_data(self, urlstring: str) -> T.Tuple[str, str]:
                 return hashvalue, tmpfile.name
             sys.stdout.flush()
             progress_bar = ProgressBar(bar_type='download', total=dlsize,
-                                       desc='Downloading')
+                                       desc='Downloading',
+                                       disable=(True if self.silent else None))
```

Comment:
Personally, I prefer like this:
```suggestion
                                       disable=(self.silent or None))
```

but I'm not sure what is the "meson" style for that.

Reply:
Indeed, done.